imap_processing/hi/l1c/hi_l1c.py

```diff
@@ -102,15 +102,15 @@ def allocate_pset_dataset(n_esa_steps: int, sensor_str: str):
     # preallocate coordinates xr.DataArrays
     coords = dict()
     # epoch coordinate has only 1 entry for pointing set
-    attrs = cdf_manager.get_variable_attributes("hi_pset_epoch").copy()
+    attrs = cdf_manager.get_variable_attributes("hi_pset_epoch", False).copy()
```

Comment:
nitpick - but - when calling keywords like this, I prefer to have it like `check_schema=False` so someone glancing through this code can see exactly what is going on.
```suggestion
    attrs = cdf_manager.get_variable_attributes("hi_pset_epoch", check_schema=False).copy()
```